src/pages/Product/Product.tsx

```diff
@@ -1,5 +1,25 @@
+import './Product.css';
+// We will need to replace with our actual products
+import { products } from '../../data';
+import ProductCard from '../../components/ProductCard/ProductCard';
+
 function Product() {
-  return <h1>Product</h1>;
+	return (
+		<>
+			<h1>Products</h1>
+			<div className='products-grid'>
+				{products.map((product) => (
+					<ProductCard
+						key={product.id}
```

Comment:
I think it's not working with the cart because I don't think you can send key as a prop. I sent the product id as an id prop and each Product card has to have an individual key. 

```
<ProductCard
            key={product.id}
            id={product.id}
            productName={product.name}
            productCategory={product.category}
            productPrice={product.price}
            productImg={product.image}
          />
```

Reply:
Thanks Alex, I've fixed that and now the addToCart buttons are working again :)